### PR TITLE
docs: update how Bytebase stores its own data and update the disaster recovery section

### PR DESCRIPTION
**Pull request by tianzhou**

### CODE REVIEW

**Review by boojack — APPROVED** [review]
LGTM

File: content/docs/en/operating/disaster-recovery.md [hunk 1]
```diff
@@ -4,8 +4,10 @@ title: Disaster Recovery
 
 ## **Periodically snapshot the entire** [**--data**](/docs/reference/command-line#data-less-than-less-than-directory-greater-than-greater-than) **directory**
 
-Bytebase stores its own data into a [SQLite](http://sqlite.com) file called `bytebase.db`under the [--data](/docs/reference/command-line#data-less-than-less-than-directory-greater-than-greater-than) directory. Besides the main data file, SQLite also creates 2 additional files: `bytebase.db-shm` for managing share memory and `bytebase.db-wal` for write ahead logging.
+<hint-block type="info">
 
-You should just periodically backup the entire [--data](/docs/reference/command-line#data-less-than-less-than-directory-greater-than-greater-than) directory. If you want to backup the data when Bytebase is running and is NOT running with [--readonly](/docs/reference/command-line#--readonly), then the underlying data volume must support snapshot feature where the entire directory can take a snapshot at the same time, otherwise it may produce a corrupted backup file.
+You should periodically backup the entire [--data](/docs/reference/command-line#data-less-than-less-than-directory-greater-than-greater-than) directory.
```
Comment (dragonly) [inline content/docs/en/operating/disaster-recovery.md:9]:
```suggestion
You should periodically backup the entire [--data](/docs/reference/command-line#--data-directory) directory.
```

Tested here: https://www.bytebase.com/docs/reference/command-line#--data-directory

Comment (dragonly) [reply content/docs/en/operating/disaster-recovery.md:9]: https://www.bytebase.com/docs/reference/command-line#data-less-than-less-than-directory-greater-than-greater-than seems not to point to a valid anchor.

File: content/docs/en/operating/how-bytebase-stores-data.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+title: How Bytebase Stores its Own Data
+---
+
+Bytebase manages its own data under the [--data](/docs/reference/command-line#data-less-than-less-than-directory-greater-than-greater-than) directory. The data includes:
```
Comment (tianzhou) [inline content/docs/en/operating/how-bytebase-stores-data.md:5]:
```suggestion
Bytebase manages its own data under the [--data](/docs/reference/command-line#--data-directory) directory. The data includes:
```

File: content/docs/en/operating/disaster-recovery.md [hunk 3]
```diff
@@ -4,8 +4,10 @@ title: Disaster Recovery
 
 ## **Periodically snapshot the entire** [**--data**](/docs/reference/command-line#data-less-than-less-than-directory-greater-than-greater-than) **directory**
```
Comment (tianzhou) [inline content/docs/en/operating/disaster-recovery.md:5]:
```suggestion
## **Periodically snapshot the entire** [**--data**](/docs/reference/command-line#--data-directory) **directory**
```

**Comment by tianzhou** [comment]
PTAL

**Review by adela-bytebase — APPROVED** [review]
LGTM

**Pull request merged**